samples/dependency-injection/msdependencyinjection/sample.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Asp.NET Core Dependency Injection Integration
+title: ASP.NET Core Dependency Injection Integration
```

Comment:
If you're going to move the URL you need to provide a redirect.

Reply:
Hu? That is just a name. I did not move anything here :)

Reply:
I was just trying to get the comment close to the metadata where it needs to be fixed.

By renaming the directory you're moving the URL from `/samples/dependency-injection/aspnetcore/` to `/samples/depencency-injection/msdependencyinjection/` but you're not providing a redirect. The first URL will result in a 404.

Honestly though I don't think you should change the URL at all. Use of the msdependencyinjection package is just an implementation detail, in fact it's showing how to do it with that package OR with autofac. The sample is still showing DI in aspnetcore, is it not? That's what the title still says.

Reply:
Ah I missed the fact that we already had such an example. That was an oversight from me. That's why I got confused because I thought there is nothing to redirect to. My thinking was that most of the samples follow a component oriented approach that's why a renamed the folder (under the wrong impression it was newly created). But given we already had it with autofac I agree with you @davidboike that the msdependencyinjection part is just another flavour of it. Will change tomorrow. Thanks for showing me where my brain made weird twists